dynamic-property-zk-test/src/main/java/ru/fix/dynamic/property/zk/test/ZKTestingServer.java

```diff
@@ -0,0 +1,139 @@
+package ru.fix.dynamic.property.zk.test;
+
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.retry.ExponentialBackoffRetry;
+import org.apache.curator.test.InstanceSpec;
+import org.apache.curator.test.TestingServer;
+import org.junit.jupiter.api.extension.BeforeTestExecutionCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.slf4j.Logger;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.UUID;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+
+public class ZKTestingServer implements BeforeTestExecutionCallback /*extends ExternalResource*/ {
+
+    private static final Logger log = getLogger(ZKTestingServer.class);
+
+    private TestingServer zkServer;
+    private Path tmpDir;
+    private int port;
+
```

Comment:
Will be in jfix-zookeeper library that @AnonymousVovus currently building

Reply:
add after that will be done